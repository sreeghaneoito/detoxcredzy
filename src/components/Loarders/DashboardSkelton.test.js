import React from 'react';
import renderer from 'react-test-renderer';
import DashboardSkelton from './DashboardSkelton.component';

test('renders correctly', () => {
  const animatedmodal = renderer.create(<DashboardSkelton />).toJSON();
  expect(animatedmodal).toMatchSnapshot();
});
