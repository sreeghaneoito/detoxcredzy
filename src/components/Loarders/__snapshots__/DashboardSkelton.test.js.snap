// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginVertical": 10,
      "paddingHorizontal": 20,
      "width": "100%",
    }
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#fff",
          "borderRadius": 16,
          "height": 140,
          "opacity": 0.6,
          "paddingVertical": 20,
          "width": "31%",
        },
        Object {
          "shadowColor": "#00000014",
          "shadowOffset": Object {
            "height": 4,
            "width": -4,
          },
          "shadowOpacity": 1,
          "shadowRadius": 24,
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#fff",
          "borderRadius": 16,
          "height": 140,
          "opacity": 0.6,
          "paddingVertical": 20,
          "width": "31%",
        },
        Object {
          "shadowColor": "#00000014",
          "shadowOffset": Object {
            "height": 4,
            "width": -4,
          },
          "shadowOpacity": 1,
          "shadowRadius": 24,
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#fff",
          "borderRadius": 16,
          "height": 140,
          "opacity": 0.6,
          "paddingVertical": 20,
          "width": "31%",
        },
        Object {
          "shadowColor": "#00000014",
          "shadowOffset": Object {
            "height": 4,
            "width": -4,
          },
          "shadowOpacity": 1,
          "shadowRadius": 24,
        },
      ]
    }
  />
</View>
`;
